### PR TITLE
Feature 1: Creating Modules

### PR DESCRIPTION
**Pull request by Dangelo-JAN**
## Creating modules for better aplication

In this feature I:

I split the app,js in two files: index.js and booklist.js

### CODE REVIEW

File: modules/booklist.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      if (id === books[i].id) {
+        books.splice(i, 1);
+      }
+    }
+
+    localStorage.setItem('books', JSON.stringify(books));
+    window.location.reload();
+  };
+
+  loadScreen() {
```
Comment (TedLivist) [inline modules/booklist.js]: - [x] Kindly change this to an arrow function.

File: modules/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import BookList from './booklist';
+
+const book = new BookList();
+document.getElementById('form').addEventListener('submit', book.add);
+book.loadScreen();
+
+const dt = new Date();
+document.getElementById('date-time').innerHTML = dt;
```
Comment (TedLivist) [inline modules/index.js]: - [x] Here you are getting your date using the vanilla Javascript, however you are  expected to render your date using the `luxon` npm package instead, kindly do that

- [x] Your page is missing the date below the navigation bar as presented in the images. Kindly fix that.

File: modules/index.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    contact.classList.replace('show', 'hide');
+  }
+});
+document.getElementById('contact-a').addEventListener('click', () => {
+  if (!contact.classList.contains('show')) {
+    list.classList.replace('show', 'hide');
+    addNew.classList.replace('show', 'hide');
+    contact.classList.replace('hide', 'show');
+  }
+});
```
Comment (TedLivist) [inline modules/index.js]: - [x] Your app displays nothing and does not work and neither does its navigations. I am unable to see anything when I navigate. Kindly fix this.

  Homepage | Add book section | Contact section
  :---:|:---:|:---:
  dangelo-1 | dangelo-2 | dangelo-3

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Dangelo-JAN,

Good job so far!
✔️ `index.js` file is present
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] **In your README documentation, kindly add more specific instructions on how to set up and run the app locally**

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Dangelo-JAN ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 




### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 
- best es6 syntax 💪🏻 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**